worlds/zork_grand_inquisitor/test/test_access.py

```diff
@@ -0,0 +1,2927 @@
+from typing import List
+
+from . import ZorkGrandInquisitorTestBase
+
+from ..enums import (
+    ZorkGrandInquisitorEvents,
+    ZorkGrandInquisitorItems,
+    ZorkGrandInquisitorLocations,
+    ZorkGrandInquisitorRegions,
+)
+
+
+class AccessTestRegions(ZorkGrandInquisitorTestBase):
+    options = {
+        "start_with_hotspot_items": "false",
+    }
+
+    def test_access_crossroads_to_dm_lair_sword(self) -> None:
```

Comment:
tests overall are fine. worth considering combining a lot of these methods at some point because each method constructs an entirely new multiworld and calls all the steps up to pre_fill.

Reply:
Noted. But my world's entire test suite runs in ~5 seconds. If that's acceptable, I would rather not butcher them recombine them in a more optimized way that won't be as explicit to save a second.